packages/loopring_v3/contracts/aux/bridge/Bridge.sol

```diff
@@ -227,11 +239,11 @@ contract Bridge is IBridge, BatchDepositor, Claimable
             transfersData := sub(transfersData, add(32, mul(34, mload(amounts))))
         }
         // Check if these transfers can be processed
-        bytes32 hash = _hashTransfers(transfersData);
-        require(!_arePendingDepositsTooOld(batch.batchID, hash), "BATCH_DEPOSITS_TOO_OLD");
+        bytes32 hash = _hashTransferBatch(batch.batchID, transfersData);
```

Comment:
@Brechtpd , is this method to process the inbound transfers (users -> bridge)? If so, when is the corresponding hash be saved?

Reply:
Any time a batch deposit is done, either from externally or internally by connectors, in `_batchDeposit`.